### DISCUSSION TITLE
Latest release with bug fixes

### DISCUSSION THREAD

**Discussion by joe-sydney** [arendst/Tasmota#14928] [Q&A] [answered]
I understand the latest "release" (e.g. 11.0.0) is available http://ota.tasmota.com/tasmota/release/.
Where do I find subsequent bug fixes? I have found the location where the development binaries are kept, eg 11.0.0.2, but are these forever just kept as development, or are they at some point moved to the main release URL and superseding the first release (ie 11.0.0)?

**Comment by ascillato2** [comment, marked as answer]
Yes, the development version is at http://ota.tasmota.com/tasmota. There you find latest bug fixes. When a release is made, the latest development version becomes the new release version.

So, when a release is made, both branches (release and development) are the same. Then, bug fixes and new features are added to the development branch. When the new release is made, the latest development is published as release, and again both branches are the same.
